src/navigation/navigation-events.js

```diff
@@ -97,7 +97,9 @@ function patchedUpdateState(updateState, methodName) {
     const urlAfter = window.location.href;
 
     if (!urlRerouteOnly || urlBefore !== urlAfter) {
-      urlReroute(createPopStateEvent(window.history.state, methodName));
+      window.dispatchEvent(
+        createPopStateEvent(window.history.state, methodName)
```

Comment:
See [this code](https://github.com/single-spa/single-spa/pull/651/files#diff-3175976bbaab1ca9a9da8a5a1dc56506e613f88d374a5c2a69da160f5d282b2dR131) which shows that firing a popstate event on the window ends up calling `urlReroute`. So this implementation is (I think) fully equivalent to the old one, and all the tests passing seem to support that.